src/main/resources/com/hcl/appscan/jenkins/plugin/scanners/StaticAnalyzer/help-openSourceOnly_en.properties

```diff
@@ -1,4 +1,4 @@
 #NLS_MESSAGEFORMAT_ALL 
 #NLS_ENCODING=UTF8
 
-description.open.source.only.scan=To run the static analysis with open source only option , the user must have this option enabled in the cloud subscription.
\ No newline at end of file
+description.open.source.only.scan=To run open source-only static analysis, ensure Software Composition Analysis is enabled in the AppScan on Cloud subscription.<br/><b>Note:</b> Software Composition Analysis is available in AppScan on Cloud only.
```

Comment:
Can we update this message to something like: "This option requires an AppScan on Cloud subscription with Software Composition Analysis (SCA) enabled."